src/test/java/org/jabref/gui/preferences/externalfiletypes/ExternalFileTypesTabViewModelTest.java

```diff
@@ -0,0 +1,85 @@
+package org.jabref.gui.preferences.externalfiletypes;
+
+import javafx.collections.ObservableList;
+
+import org.jabref.gui.DialogService;
+import org.jabref.preferences.FilePreferences;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Spy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+
+public class ExternalFileTypesTabViewModelTest {
+
+    private FilePreferences filePreferences = mock(FilePreferences.class);
+    private DialogService dialogService = mock(DialogService.class);
+    @Spy
+    private ExternalFileTypesTabViewModel externalFileTypesTabViewModel = spy(new ExternalFileTypesTabViewModel(filePreferences, dialogService));
+    private ExternalFileTypeItemViewModelTestData externalFileTypeItemViewModel = new ExternalFileTypeItemViewModelTestData();
+   @BeforeEach
+   void setUp() {
+       // arrange
```

Comment:
Indent is off.

Please no `// arrange`, `action, assert` comments throughout the test code. That all should be obvious.

Reply:
comment removed thanks